[Cleanup] General Table Styling Adjustment

Civolilah opened the pull request:
@beganovich @turbo124 The PR includes final styling of the data table along with adjusting the border color on the dashboard to match Figma. Screenshots:

![Screenshot 2025-04-01 at 19 18 22](https://github.com/user-attachments/assets/259ec32f-b5f5-44ed-ab42-ac84b41fe83f)

![Screenshot 2025-04-01 at 19 19 04](https://github.com/user-attachments/assets/79c93ddb-5cf7-442b-8610-fac4ee1787ac)

Let me know your thoughts.

turbo124 commented:
this looks good to me, @beganovich could you run your eyes over it also please?

beganovich commented:
Couple issues with this:

1. Not related to this PR, but background colour doesn't seem to match, creating issues with contrast. 
2. Contrast is very low. I've checked colours, seems to match, but the problem is font weight. Make sure to match that both in table PRs, and please fix it for the badges too.
3. Table roundness.

Civolilah commented:
> Couple issues with this:
> 
> 1. Not related to this PR, but background colour doesn't seem to match, creating issues with contrast.
> 2. Contrast is very low. I've checked colours, seems to match, but the problem is font weight. Make sure to match that both in table PRs, and please fix it for the badges too.
> 3. Table roundness.

@beganovich Thanks for the advice. Yes, the background color did not match, so I made the color match and it improved contrast. But between background color fields and table, the issue was the wrong border color. I checked Figma and found that the border color was wrong (it just needed to be a little bit darker.), so I've fixed it also. Table roundness is also fixed. Let me know if I fixed anything.

beganovich reviewed (approved):
Looks good, thanks :+1: 

I'd imagine we might get some users to complain about contrast and muted/pastel colors, but let's roll and see if they like it.
